tools/organization-cleanup/pipeline.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+echo "no pipeline required, as this is testing in pipeline-runner"
```

Comment:
Can we please not do this in order to have a clean output during pipeline executions? This can be a comment in this file if you like without producing any outputs?